auth-web/src/models/request-tracker.ts

```diff
@@ -0,0 +1,14 @@
+export interface RequestTracker {
+  id: number
+  requestType: string
```

Comment:
Could an enum be used for this?

Reply:
We could consider this in the next ticket, when we show the list of request

Reply:
OK, please add a todo checkbox to the next ticket.